### PR TITLE
Added Soup

### PR DESCRIPTION
**Pull request by Vault108-zz**
Added a soup Command.

### CODE REVIEW

File: data/food/soup.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    "templates": [
+        "hands {user} a {soup} soup."
+    ],
+    "parts": {
+        "soup": [
+" Bowl of Ajiaco ",
```
Comment (linuxdaemon) [inline data/food/soup.json]: The leading space in this string will result in an output of `hands <user> a  Bowl of Ajiaco ` (note the double space before Bowl and the trailing whitespace). I'd recommend removing surrounding whitespace from these strings.
I'd also recommend running the `format_json.py` script from the repository root to reformat the JSON file to follow the standard format for this repository.

File: data/food/soup.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+" Bowl of Gomguk ",
+" Bowl of Goulash soup ",
+" Bowl of Gumbo",
+" Bowl of  and sour soup ",
+" Bowl of Kharcho",
+" Bowl of Kimchi Guk ",
+" Bowl of Lagman",
+" Bowl of Leek soup",
+" Bowl of Lentil soup",
+" Bowl of Matzah ball soup",
```
Comment (linuxdaemon) [inline data/food/soup.json]: These options would result in `hands <user> a Bowl of Matzah ball soup soup` due to the fact that the template itself adds the word 'soup'

File: plugins/foods.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
     BasicFood("beer", "beer", "beer.json", "beer"),
     BasicFood("cheese", "cheese", "cheese.json", "cheese"),
     BasicFood("pancake", "pancake", "pancake.json", "pancakes"),
+    BasicFood("soup","soup","soup.json","soup"),
```
Comment (linuxdaemon) [inline plugins/foods.py]: There should be a single space following a comma in a parameter list

File: data/food/soup.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+"Bowl of Miso",
+"Bowl of She-crab",
+"Bowl of Tomato",
+"Bowl of Tteokguk",
+"Bowl of Winter mellon",
+"Bowl of Crab Gaxpacho",
+"Bowl of Salmorej",
+"Bowl of Tarator",
+"Bowl of New England clam chowder"
+]
```
Comment (linuxdaemon) [inline data/food/soup.json]: This indentation should be fixed using the `format_json.py` script

File: data/food/soup.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
+{
+    "templates": [
+        "hands {user} a {soup} soup."
+    ],
+    "parts": {
+        "soup": [
+"Bowl of Ajiaco ",
+"Bowl of Avgolemono ",
+"Bowl of Borscht ",
```
Comment (linuxdaemon) [inline data/food/soup.json]: The trailing whitespace here should be removed as well

**Review by linuxdaemon — CHANGES_REQUESTED** [review]

**Comment by edwardslabs** [comment]
There are merge conflicts now because I took too long to merge several PRs in. Please fix the merge conflicts.

**Comment by Vault108-zz** [comment]
Moved to #220

**Pull request closed without merging**